src/app/(private)/(routes)/admin/courses/[courseId]/components/new-course-form.tsx

```diff
@@ -37,6 +37,7 @@ import * as z from 'zod'
 type NewCourseFormProps = {
   initialData: Course | null
   categories: Category[]
+  feedback: boolean
```

Comment:
Seguindo os padrões de clean code, geralmente atributos booleanos inserimos um "verb to be" (como no uso do "isPending" que vc fez) a depender do contexto e do uso da variavel, nesse caso acho ideal renomear essa variavel para "hasFeedback"